include/smack/BoogieAst.h

```diff
@@ -306,8 +306,9 @@ class CallStmt : public Stmt {
 
 class Comment : public Stmt {
   std::string str;
+  bool star;
 public:
-  Comment(std::string s) : Stmt(COMMENT), str(s) {}
+  Comment(std::string s, bool st) : Stmt(COMMENT), str(s), star(st){}
```

Comment:
Why do you need `star` here?
Also, if this is needed, it should be renamed to something like `blockComment`.

Reply:
This may have occurred during a messy merge. I took the file from develop.